opencensus/stats/exporters/prometheus_exporter.py

```diff
@@ -164,31 +164,13 @@ def to_metric(self, desc, view):
                                        labels=labels)
         elif isinstance(agg_data,
                         aggregation_data_module.DistributionAggregationData):
+
+            assert(agg_data.bounds == sorted(agg_data.bounds))
```

Comment:
We should do same in the stats/stackdriver exporter?

Reply:
[Here](https://github.com/census-instrumentation/opencensus-python/blob/19f89e19f167e2f0c55fbb288b228a9e4053f020/opencensus/stats/exporters/stackdriver_exporter.py#L221-L222)? I don't think we need this assertion everywhere we use distribution aggregations, I've only got it in the prometheus exporter since guaranteeing the bounds were sorted let me simplify the code a bit.